### PR TITLE
Ports RDX and TaTP from /tg/

### PR DESCRIPTION
**Pull request by totallynotkatfood**
## About The Pull Request

big boom chems go boom

## Why It's Good For The Game

More challenging and fun chems for the choomist to make

## Changelog
:cl:
add: Added RDX and TaTP
add: Added component chems for RDX and TaTP
/:cl:

### CODE REVIEW

**Comment by PapaSejd** [comment]
Based

**Comment by YakiAttaki** [comment]
hell yes

File: code/game/objects/structures/lavaland/geyser.dm [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
 /obj/structure/geyser/random
 	erupting_state = null
-	var/list/options = list(/datum/reagent/oil = 2, /datum/reagent/clf3 = 1) //fucking add more
+	var/list/options = list(/datum/reagent/oil = 2, /datum/reagent/clf3 = 1, /datum/reagent/plasma_oxide = 8) //fucking add more //austation -- RDX, TATP
```
Comment (TheFakeElon) [inline code/game/objects/structures/lavaland/geyser.dm:47]: plasma oxide*

File: code/game/objects/structures/lavaland/geyser.dm [hunk 2]
```diff
@@ -65,7 +65,7 @@
 
 /obj/item/plunger/reinforced
 	name = "reinforced plunger"
-	desc = " It's an M. 7 Reinforced Plunger� for heavy duty plunging."
+	desc = " It's an M. 7 Reinforced Plunger� for heavy duty plunging."
```
Comment (TheFakeElon) [inline code/game/objects/structures/lavaland/geyser.dm:68]: special character broke

File: code/game/objects/structures/lavaland/geyser.dm (hunk 1 above)
Comment (totallynotkatfood) [reply code/game/objects/structures/lavaland/geyser.dm:47]: yes, it’s actually named that

Comment (TheFakeElon) [reply code/game/objects/structures/lavaland/geyser.dm:47]: I’m talking about the code comment silly

Comment (totallynotkatfood) [reply code/game/objects/structures/lavaland/geyser.dm:47]: It's referencing the pr that added RDX and tatp, but whatever

File: code/game/objects/structures/lavaland/geyser.dm [hunk 3]
```diff
@@ -44,7 +44,7 @@
 
 /obj/structure/geyser/random
 	erupting_state = null
-	var/list/options = list(/datum/reagent/oil = 2, /datum/reagent/clf3 = 1) //fucking add more
+	var/list/options = list(/datum/reagent/oil = 2, /datum/reagent/clf3 = 1, /datum/reagent/plasma_oxide = 8) //fucking add more //austation -- plasma_oxide for exotic stabilizer
```
Comment (MCterra10) [inline code/game/objects/structures/lavaland/geyser.dm:47]: poggers, more lore that plasma is actually titrated ethanol

**Review by MCterra10 — APPROVED** [review]

**Pull request merged**